### PR TITLE
Added product-list page

### PR DESCRIPTION
**Pull request by shristabaruah**
--Added product list screen

### CODE REVIEW

File: product list/product.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.product-container{
+    display: flex;
+    margin: 1rem;
+}
+.filter-container{
+    position: fixed;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: .5rem;
```
Comment (starver20) [inline product list/product.css]: It will be better to write decimal numbers with a zero, makes it easier to read.

**Review by Aazam10 — APPROVED** [review]
You can make use of more semantic tags like main section in your markup rest everything looks great

File: product list/product.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    display: flex;
+    justify-content: space-between;
+}
+.center-hr{
+border-top: 1px solid var(--color-border);
+padding: .5rem;
+}
+.header{
+    font-size: 1rem;
+    letter-spacing: 1px;
```
Comment (starver20) [inline product list/product.css:24]: Can use css variables for font size and spacings.

**Review by starver20 — APPROVED** [review]
Other than suggested rest code looks good, color selection is also good.

File: product list/product.css (hunk 1 above)
Comment (shristabaruah) [reply product list/product.css]: Okay will do the changes!!

File: product list/product.css (hunk 2 above)
Comment (shristabaruah) [reply product list/product.css:24]: Yes will try to refactor the code

**Comment by shristabaruah** [comment]
> You can make use of more semantic tags like main section in your markup rest everything looks great

Will try to refactor the code !!

**Pull request merged**